CMakeLists.txt

```diff
@@ -1,16 +1,33 @@
-# Copyright © 2024 Bret Brown
-# SPDX-License-Identifier: MIT
+# /CMakeLists.txt -*-makefile-*-
```

Comment:
```suggestion
```
I'm unconvinced these file identifier lines are standard practice or worthwhile.

Reply:
Depends where you've been in industry. Having had to do code inspection reviews with things printed out means that things need self identification. There is no IDE. 

Tagging the kind of file is fairly common 1.4M hits for  "-*- C++ -*-" https://github.com/search?q=%22-*-+c%2B%2B+-*-%22&type=code  65K for "-*- Makefile -*-" 

It of course shows up in all FSF software, since emacs uses it to identify the mode for the file. Definitely an issue when you have C++ in dot h files. 

There's more divergence in file name, with in addition to just the bare name here, using \file, @file, or other markup for identifying the work under copyright. The name was originally the canonical name for the file, not a position in the repository. For headers this is useful because it documents what the spelling of the include should be. That is, it should always be spelled `#include <Beman/Optional26/optional.hpp>` not `optional.hpp` or some other form. Implementation files don't really need a canonical name, particularly if the library is not spread across multiple directories, and the CML.txt is local to the file being added to the sources list for a file. 

Yes, this particular form comes from the BDE style guide, which originated with Mentor Graphics ages ago. I've got a yasnippet macro that fills out a header and implementation when I create a new file. The guide is of course highly pedantic and detail oriented, but at least it's all written down: 
https://bloomberg.github.io/bde/knowledge_base/coding_standards.html#prologue

Reply:
I will put this on our agenda. I would like to get a consensus, and make a recommendation.

Reply:
Removed after our sync: `Allow comments for tooling, not recommended or required by the Beman Standard. (Outcome: Unanimous consent)`